readthedocs.yml

```diff
@@ -1,5 +1,5 @@
 conda:
-    file: docs/requirements.yml
+    file: devtools/conda-envs/base.yaml
```

Comment:
Doesn't doing this (and removing the requirements.yml file) mean that we now need to add all the missing dependencies to the base env file? So sphinx, sphinx_rtd_theme, sphinx-automodapi, and graphviz? I'm not sure we want to do that.

Reply:
Ah crap, yea.